base/dcp_feed.go

```diff
@@ -661,11 +689,8 @@ func StartDCPFeed(bucket Bucket, spec BucketSpec, args sgbucket.FeedArguments, c
 // version number / commit for debugging purposes
 func GenerateDcpStreamName(product string) (string, error) {
 
-	// Create a time-based UUID for uniqueness of DCP Stream Names
-	u, err := uuid.NewUUID()
-	if err != nil {
-		return "", err
-	}
+	// Create a non-time-based UUID for uniqueness of DCP Stream Names
+	u := uuid.New()
```

Comment:
This panics on error - perhaps we should use this version instead:

```suggestion
	u, err := uuid.NewRandom()
	if err != nil {
		return "", err
	}
```